chord_metadata_service/patients/api_views.py

```diff
@@ -17,6 +17,8 @@
 from rest_framework.response import Response
 from rest_framework.settings import api_settings
 
+from chord_metadata_service.chord.views_search import build_experiments_by_subject
+from chord_metadata_service.chord.views_search import get_biosamples_with_experiment_details
```

Comment:
instead of importing from views, move this function to a util in the chord package and import it from there